FeedSX/viewDataModels/postViewData/index.tsx

```diff
@@ -0,0 +1,26 @@
+export function postViewData(data: any): LMPostUI[] {
+  let postData = data?.posts;
+  let userData = data?.users;
+  return postData.map((item: any) => {
+    return {
```

Comment:
we should mention the name of the interface exported like here it should be `LMPostUI`